docker-compose-dependencies.yml

```diff
@@ -0,0 +1,76 @@
+version: '3.1'
```

Comment:
this service  already have one  docker file 'docker-compose-full.yml'

Reply:
docker-compose-full loads up manage-users-api itself. docker-compose-dependencies is there for dev purposes and doesn't load the service itself. I'll add a comment to it.

Reply:
I've removed it - I can run docker-compose-full with specific services